static/js/components/CommentForms.js

```diff
@@ -184,17 +185,18 @@ const mapDispatchToProps = (dispatch: any, ownProps: CommentFormProps) => {
           if (err.errorStatusCode === 410) {
             // Comment was deleted
             dispatch(
-              setSnackbarMessage({
-                message:
-                  "This comment has been deleted and cannot be replied to"
-              })
+              setBannerMessage(
+                "This comment has been deleted and cannot be replied to"
+              )
             )
           } else {
             // Unknown errors
             dispatch(
-              setSnackbarMessage({
-                message: "Unknown error replying to comment"
-              })
+              setBannerMessage(
+                `Something went wrong creating your comment. Please try again or contact us at ${
+                  SETTINGS.support_email
```

Comment:
should we maybe do a `mailto` link here? not sure if the banner supports that

Reply:
Gave it a try, seems like only plain text will work

Reply:
ah, we'll that's ok, sort of a 'nice to have' anyhow